forge-gui/res/cardsfolder/upcoming/overwhelming_encounter.txt

```diff
@@ -1,11 +1,10 @@
 Name:Overwhelming Encounter
 ManaCost:3 G G
 Types:Sorcery
-A:SP$ PumpAll | ValidCards$ Creature.YouCtrl | KW$ Vigilance & Trample | SubAbility$ DBRollDice | SpellDescription$ Creatures you control gain vigilance and trample until end of turn. Roll a d20.
```

Comment:
Isn't the roll d20 part of the spell description when this card is put into the stack? 
![image](https://user-images.githubusercontent.com/9781539/180609144-f2231c03-c0d8-477f-907c-ac0638abfee9.png)

Reply:
Yes, but the roll d20 part should be handled by line 5

With these changes:
![image](https://user-images.githubusercontent.com/103371817/180611847-5617a6e2-6bb3-4bc8-86dd-64bb5fc6169d.png)

Old version:
![image](https://user-images.githubusercontent.com/103371817/180611953-1a6befd3-b516-42d0-98a6-f6d129718587.png)